Orange/data/table.py

```diff
@@ -35,6 +35,12 @@ def get_sample_datasets_dir():
 dataset_dirs = ['', get_sample_datasets_dir()]
 
 
+"""Domain conversion cache used in from_table. It is global so that
```

Comment:
from_table -> Table.from_table (although it is the only such method, I would be explicit)